specs/logger.js

```diff
@@ -0,0 +1,108 @@
+const Parse = require('parse/node').Parse;
+
+// this is just a copy and paste of:
+// https://github.com/ParsePlatform/parse-server/blob/master/spec/adapter.spec.js
+var testAdapter = function (name, adapter) {
+  it("Verify INFO logs", (done) => {
+    const message = 'testing info logs';
+    adapter.info(message, () => {
+      adapter.query({
+        from: new Date(Date.now() - 500),
+        size: 100,
+        level: 'info'
+      }, (results) => {
+        if (results.length == 0) {
+          fail('The adapter should return non-empty results');
+        } else {
+          expect(results[0].message).toEqual(message);
+          done();
+        }
+      });
+    });
+  });
+
+  describe('error logs', () => {
+    it("Verify ERROR logs", (done) => {
+      const message = 'testing error logs';
+      adapter.error(message, () => {
+        adapter.query({
+          from: new Date(Date.now() - 500),
+          size: 100,
+          level: 'error'
+        }, (results) => {
+          if (results.length == 0) {
+            fail('The adapter should return non-empty results');
+            done();
+          }
+          else {
+            expect(results[0].message).toEqual(message);
+            done();
+          }
+        });
+      });
+    });
+  });
+
+//   describe('verbose logs', () => {
+//     it("mask sensitive information in _User class", (done) => {
```

Comment:
these tests would be nice to run, but i got hung up on getting reconfigure server visible and operational.  i could take another crack at it now that the whole thing is working.